### ISSUE TITLE
 The option definition `services.xserver.displayManager.gdm.nvidiaWayland' in `/nix/store/knyssisqjd0p76kyxag14c5drqf55cd6-source/hosts/desktop/config.nix' no longer has any effect; please remove it.

### ISSUE THREAD

**Issue by teto** [cole-mickens/nixcfg#5]
I am trying to fix an issue with my setup (RTX 3060) 
```
00:00:00.006 [sway/main.c:62] !!! Proprietary Nvidia drivers are in use !!!
00:00:00.006 [sway/server.c:50] Preparing Wayland server initialization
00:00:00.006 [wlr] [backend/x11/backend.c:397] Creating X11 backend
00:00:00.007 [wlr] [backend/x11/backend.c:480] X11 does not support shared pixmaps
00:00:00.007 [wlr] [backend/x11/backend.c:609] Failed to query DRI3 DRM FD
```
and so I tried to copy your config referenced at https://github.com/nix-community/nixpkgs-wayland#nvidia-users but got
```
       Failed assertions:
       - The option definition `services.xserver.displayManager.gdm.nvidiaWayland' in `/nix/store/knyssisqjd0p76kyxag14c5drqf55cd6-source/hosts/desktop/config.nix' no longer has any effect; please remove it.
       We defer to GDM whether Wayland should be enabled.
```
https://github.com/colemickens/nixcfg/blob/cdd9929d5d36ce5b4d64cf80bdeb1df3f2cba332/mixins/nvidia.nix#L36

**Comment by colemickens** [comment]
Really not sure off the top of my head. That is an old "nvidia" config snippet though, and I realized that if I link you to the new one, it won't work either since I have a rather large (rfc 121) change that my config all relies on.

AFAIK though, these days it should mostly just work. I was running this a week or two ago, not just "fine" but had Steam going playing a AAA game, etc.

**Comment by teto** [comment]
I had sway working with my previous nvidia card (a 970 GTX) but upgrading to RTX 3060 it doesn't anymore. I've tried following  https://discourse.nixos.org/t/nvidia-users-testers-requested-sway-on-nvidia-steam-on-wayland/15264/61 but the thread is now hard to follow and for instance I dont even know if I need the nixpkgs-wayl;and overlay anymore ? 

It's best I open a proper report somewhere else though, thanks for your work  and patience.
